File: src/leetcode/977. Squares of a Sorted Array/sq_sorted_array977.go
func sortedSquares(nums []int) []int {    
    result := make([]int, len(nums))
    for i := 0;  i < len(nums); i++ {
        result[i] = nums[i] * nums[i]
    }
    sort.Ints(result)
    return result
}

File: src/leetcode/977. Squares of a Sorted Array/sq_sorted_array977.cpp
class Solution {
public:
    vector<int> sortedSquares(vector<int>& nums) {
        vector<int> agNums;

        for(int i = 0; i < nums.size(); i++) {
            agNums.push_back(pow(nums[i], 2));
        }

        sort(agNums.begin(), agNums.end());

        return agNums;
    }
};
